cmd/global-heal.go

```diff
@@ -214,11 +214,7 @@ func (er *erasureObjects) healErasureSet(ctx context.Context, buckets []string,
 
 		disks, _ := er.getOnlineDisksWithHealing()
 		if len(disks) == 0 {
-			// all disks are healing in this set, this is allowed
-			// so we simply proceed to next bucket, marking the bucket
-			// as done as there are no objects to heal.
-			tracker.bucketDone(bucket)
-			logger.LogIf(ctx, tracker.update(ctx))
+			logger.LogIf(ctx, fmt.Errorf("no enough online disks found to heal bucket `%s`", bucket))
```

Comment:
This is something needed because we allow all the drives in a given "set" to fail, it logs and skips the bucket healing since we can't get any disks anymore. 

@vadmeste `make verify-healing` verifies this scenario.

Reply:
I am not sure I understand. The old code marks the bucket as done. This should not be done, since the bucket is not really healed.

Reply:
It did fix a bug but not sure what it was anymore.